server/controllers/notification.js

```diff
@@ -0,0 +1,57 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+exports.createNotification = asyncHandler(async (req, res) => {
+    const {
+        type,
+        description,
+        read,
+        createdBy,
```

Comment:
```suggestion
```
Rather than sending the creator's id in the request body, why don't we obtain it from the logged in user (look up profile using their id from `req.user.id`) for additional security - this can be fixed in a separate PR with the other BE related changes

Reply:
Addressed in new commit.